msteams-platform/TOC.yml

```diff
@@ -368,6 +370,16 @@
     href: graph-api/import-messages/import-external-messages-to-teams.md
   - name: Code samples
     href: https://github.com/OfficeDev/microsoft-teams-samples#graph-apis
+- name: Extend your app across Microsoft 365
```

Comment:
Hi @erikadoyle,

Please add a paragraph in the Build apps for Teams page:

 https://docs.microsoft.com/en-us/microsoftteams/platform/overview

Reply:
Added in next commit

Reply:
How does this look: https://review.docs.microsoft.com/en-us/microsoftteams/platform/overview?branch=pr-en-us-4554#test-your-app-running-across-microsoft-365